public/app/ui/components/collections/library/thumbnails-result.html

```diff
@@ -0,0 +1,6 @@
+<div class="thumbnail-result thumbnail-result-library">
```

Comment:
This is the per-collection view of each result in "Thumbnails" mode.  I don't expect there to be much variation between the collections here, if any, but doing it this way for consistency.